packages/teuchos/parameterlist/src/Teuchos_XMLObjectImplem.cpp

```diff
@@ -210,7 +210,7 @@ std::string XMLObjectImplem::toString() const
   std::string rtn;
   if (content_.length()==0 && children_.length()==0)
   {
-    rtn = terminatedHeader() + "\n";
+    rtn = terminatedHeader(true) + "\n";
```

Comment:
This makes this consistent with:

https://github.com/cgcgcg/Trilinos/blob/00ef95f4fd641f824502e499eca5c833fd354210/packages/teuchos/parameterlist/src/Teuchos_XMLObjectImplem.cpp#L249

so this seems logical.

Not sure why you would **not** want to terminate using strict XML syntax.  What is that use case for?